packages/cli/src/lib/cmds/init.ts

```diff
@@ -245,6 +246,10 @@ export async function init(
     twaManifest.isChromeOSOnly = true;
   }
 
+  if (args.metaquest) {
```

Comment:
I'm wondering if passing `--chromeosonly` and `--metaquest` together should throw an error, or maybe a warning, since setting Chrome OS only will effectively make the APK incompatible with Meta Quest.

Reply:
The `--chromeosonly` and `--metaquest` options should be fully compatible. Meta Quest uses only the `AndroidManifest.xml` from the APK file. All unknown things in `AndroidManifest.xml` are simply ignored. It seems to be the same for Chrome OS. But I can check again to be sure.

Reply:
Chrome OS only adds `<uses-feature android:name="org.chromium.arc" android:required="true" />`. Since it contains `required=true`, in theory, app stores should only install the package on Chrome OS devices. No clue if other stores are handling this though

Reply:
I've double-checked, APK built with `--chromeosonly` and `--metaquest` options together installs and works on Meta Quest like a charm.

Also I've tried `ovr-platform-util upload-quest-build` command on it. The validator doesn't throw any errors about `<uses-feature android:name="org.chromium.arc" android:required="true" />`. It seems App Lab and Oculus Quests just ignore it.

I think increasing `minSdkVersion` form 19 to 23 doesn't break Chrome OS compatibility. Right?

Reply:
In this case, `uses-feature` is used the application stores to figure out if the application is compatible with a given device. If it isn't it would be installable from the app store. I don't think increasing minSdkVersion will break compatibility.